lib/src/Uncertainty/Algorithm/MetaModel/LinearModel/LinearModelAnalysis.cxx

```diff
@@ -432,7 +430,7 @@ Graph LinearModelAnalysis::drawResidualsVsFitted() const
   Cloud cloud(dataFull, "black", "fcircle");
   graph.add(cloud);
   // Add point identifiers for worst residuals
-  UnsignedInteger identifiers(ResourceMap::GetAsUnsignedInteger("LinearModelAnalysis-Identifiers"));
+  UnsignedInteger identifiers(ResourceMap::GetAsUnsignedInteger("LinearModelAnalysis-IdentifiersNumber"));
```

Comment:
Why don't you take the minimum between size and this ResourceMap key, as in drawModelVsFitted?

Reply:
That is a bug: good catch!